data-table/table.js

```diff
@@ -43,6 +144,47 @@ function createEntryCell(indexString) {
      */
 }
 
+function cellIndexToElementId(rowIndex, colIndex) {
+    return "_row_" + rowIndex + "_and_col_" + colIndex + "_";
```

Comment:
be sure to include `wrapperDivId` in this string, otherwise you can't have multiple datatables on the same page